src/Recordings.cpp

```diff
@@ -607,6 +607,11 @@ PVR_ERROR Recordings::GetRecordingEdl(const kodi::addon::PVRRecording& recording
     tinyxml2::XMLNode* pCommercialNode;
     for (pCommercialNode = commercialsNode->FirstChildElement("commercial"); pCommercialNode; pCommercialNode = pCommercialNode->NextSiblingElement())
     {
+      if (index++ == PVR_ADDON_EDL_LENGTH)
+      {
```

Comment:
I think this deserves a comment explaining why we should break when the condition is met.

Reply:
Ok I added some comments I linked to Kodi master but Matrix as a similar issue.